pyClient/zeth/constants.py

```diff
@@ -66,45 +97,22 @@
 #   [2 + jsOut + 2*jsIn] - 1     x residual bits, v_in, v_out
 
 # Index (in public inputs) of residual bits
-RESIDUAL_BITS_INDEX = (2 * JS_INPUTS) + JS_OUTPUTS + 2
+RESIDUAL_BITS_INDEX: int = (2 * JS_INPUTS) + JS_OUTPUTS + 2
 
 # Number of full-length digests to be encoded in public inputs
-NUM_INPUT_DIGESTS = (2 * JS_INPUTS) + 1
+NUM_INPUT_DIGESTS: int = (2 * JS_INPUTS) + 1
 
 # Total number of residual bits corresponding to digests in public inputs
-TOTAL_DIGEST_RESIDUAL_BITS = NUM_INPUT_DIGESTS * DIGEST_RESIDUAL_BITS
+TOTAL_DIGEST_RESIDUAL_BITS: int = NUM_INPUT_DIGESTS * DIGEST_RESIDUAL_BITS
 
 # Solidity compiler version
-SOL_COMPILER_VERSION = 'v0.5.16'
+SOL_COMPILER_VERSION: str = 'v0.5.16'
 
 # Seed for MIMC
 MIMC_MT_SEED: str = "clearmatics_mt_seed"
 
 # Units for vpub_in and vpub_out, given in Wei. i.e.
 #   Value (in Wei) = vpub_{in,out} * ZETH_PUBLIC_UNIT_VALUE
-ZETH_PUBLIC_UNIT_VALUE = 1000000000000  # 1 Szabo (10^12 Wei).
-
-COMMITMENT_VALUE_PADDING = bytes(int(192/8))
-
-# Key Derivation Tag "ZethEnc" utf-8 encoding
-KDF_TAG: bytes = b'ZethEnc'
-
-# Note constants
-APK_LENGTH: int = 256
-NOTE_VALUE_LENGTH: int = 64
-RHO_LENGTH: int = 256
-TRAPR_LENGTH: int = 256
+ZETH_PUBLIC_UNIT_VALUE: int = 1000000000000  # 1 Szabo (10^12 Wei).
 
-# Encryption constants length in bits
-EC_PRIVATE_KEY_LENGTH: int = 256
-EC_PUBLIC_KEY_LENGTH: int = 256
-SYM_KEY_LENGTH: int = 256
-MAC_KEY_LENGTH: int = 256
-TAG_LENGTH: int = 128
-KEY_MATERIAL: int = SYM_KEY_LENGTH + MAC_KEY_LENGTH
-SYM_NONCE_LENGTH: int = 128
-NOTE_LENGTH: int = APK_LENGTH + NOTE_VALUE_LENGTH + RHO_LENGTH + TRAPR_LENGTH
-ENCRYPTED_NOTE_LENGTH: int = EC_PUBLIC_KEY_LENGTH + NOTE_LENGTH + TAG_LENGTH
-
-# Encryption constants values
-SYM_NONCE_VALUE: int = 0
+COMMITMENT_VALUE_PADDING: bytes = bytes(int(192/8))
```

Comment:
Let's add a comment here. Using numbers (without labels) like here can be confusing. Here 192 comes from the fact that the value of the notes is encoded on 64 bits, and we need to pad it by 192 bits to get the other half of the input block for blake2s (which we use as commitment).

Do we actually use this constant somewhere else? I believe that we shouldn't because we do not call the compression function multiple times manually but rather switched to use the full hash function for the commitment, so we shouldn't need this padding anymore

Reply:
Yes, it seems unused now.  Removed.